packages/components/transitions/src/fade.tsx

```diff
@@ -42,44 +42,37 @@ export const fadeProps = {
  *
  * @see Docs https://yamada-ui.com/components/transitions/fade
  */
-export const Fade = forwardRef<FadeProps, "div", false>(
-  (
-    {
-      unmountOnExit,
-      isOpen,
-      transition,
-      transitionEnd,
-      delay,
-      duration,
-      className,
-      ...rest
-    },
-    ref,
-  ) => {
-    const animate = isOpen || unmountOnExit ? "enter" : "exit"
+export const Fade = forwardRef<FadeProps, "div", false>((props, ref) => {
+  const [style, mergedProps] = useComponentStyle("Fade", props)
+  const {
+    unmountOnExit,
+    transition,
+    transitionEnd,
+    delay,
+    duration,
+    className,
+    ...rest
+  } = omitThemeProps(mergedProps)
 
-    const custom = { transition, transitionEnd, delay, duration }
+  const animate = rest.isOpen || unmountOnExit ? "enter" : "exit"
 
-    isOpen = unmountOnExit ? isOpen && unmountOnExit : true
+  const custom = { transition, transitionEnd, delay, duration }
 
-    const css: CSSUIObject = {
-      w: "100%",
-    }
+  const isOpen = unmountOnExit ? rest.isOpen && unmountOnExit : true
```

Comment:
Why was it necessary to modify this section?If you don't need to fix it, please revert back to the original.

Also, this modification will cause a warning on the console because of the `isOpen` in `rest`.